### DISCUSSION TITLE
Python code support?

### DISCUSSION THREAD

**Discussion by anquanscan** [projectdiscovery/nuclei#4052] [Q&A]
Hello, I'd like to ask which version currently supports Python code? I saw in your official documentation that it supports writing YAML using Python code, but when I tried it, it didn't work.

**Comment by ehsandeep** [comment]
@anquanscan it's in the `v3-beta` branch.

https://github.com/projectdiscovery/nuclei/tree/v3-beta

**Reply by anquanscan** [reply]
When is the expected release date for v3-beta?

**Reply by ehsandeep** [reply]
As soon as below (WIP) PRs get merged.

https://github.com/projectdiscovery/nuclei/pull/3981
https://github.com/projectdiscovery/nuclei/pull/4015

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
